README.md

```diff
@@ -1,2 +1,98 @@
-# todo-list
-Simple todo list for study purpusess
+
+# FastAPI Application
+
+A basic FastAPI application set up with test endpoints interacting with a PostgreSQL database running on a docker container. 
+
+## Setup
+
+### Step 1: Create and activate the Virtual Environment
+
+To create and enter the virtual environment, run:
+
+```bash
+python3 -m venv todo_list
+source todo_list/bin/activate 
+```
+### Step 2: Install the dependencies
+
+Install the dependencies from the `requirements.txt` file:
+
+```bash
+pip install -r requirements.txt 
+```
+### Step 3: Configure ENV variables
+
+Create a file ".env" with the following variables:
+
+```bash
+host=HOST  
+database=DATABASE  
+user=DB_USER  
```

Comment:
Usually, it\'s better to add the db_ prefix to host, username, and password variablse so it will be apparent that we are talking about connections to the database. Also, if in the future we add additional external systems, we could differentiate between the password for the database and other systems.